addOns/postman/src/main/java/org/zaproxy/addon/postman/models/KeyValueData.java

```diff
@@ -38,10 +38,10 @@ public KeyValueData(String key, String value) {
     }
 
     @JsonDeserialize(using = ObjectDeserializer.class)
-    private String key;
+    private String key = "";
```

Comment:
This alone does not ensure that the string will be non-null, the user can explicitly set the field to null and it will end up being null here. For that it would have to be added something like `@JsonSetter(nulls = Nulls.SKIP)` too.